database_management_bot.py

```diff
@@ -0,0 +1,2235 @@
+                reply_keyboard,
+                one_time_keyboard=True,
+                resize_keyboard=True,
+                input_field_placeholder='Оберіть опцію',
+            ),
+        )
+    except Exception as e:
+        logger.warning("Something went wrong during review")
+        logger.warning(e)
+
+        update.message.reply_text(
+            text="*Упс\.\.\. Щось пішло не так* 😞️"
+                 "\n\nСпробуйте ще раз, або звʼяжіться з адміністратором бота\.",
+            parse_mode="MarkdownV2",
+            reply_markup=ReplyKeyboardMarkup(
+                reply_keyboard,
+                one_time_keyboard=True,
+                resize_keyboard=True,
+                input_field_placeholder='Оберіть опцію',
+            ),
+        )
+
+    return ConversationHandler.END
+
+
+@under_maintenance
+@superuser
+def statistics_for_user(update: Update, context: CallbackContext) -> int:
+    """
+    The statistics_for_user function is used to get statistics for a specific user.
+    It takes the update and context objects as parameters, then asks the user to input an ID of a user.
+
+    :param update: Update: Access the context of the conversation
+    :param context: CallbackContext: Pass data between the callback functions
+    :return: The user's id
+    """
+    reply_keyboard = [['Скасувати']]
+
+    logger.info("/statistics. Asking for the ID")
+
+    update.message.reply_text(
+        text="Введіть ID користувача для отримання статистики",
+        reply_markup=ReplyKeyboardMarkup(
+            reply_keyboard,
+            one_time_keyboard=True,
+            resize_keyboard=True,
+            input_field_placeholder='Оберіть опцію',
+        ),
+    )
+    return STATISTICS
+
+
+def get_admin_info(user_id):
+    """
+    The get_admin_info function returns the formatted admin information for a given user_id.
+
+    :param user_id: Find the user in the admins collection
+    :return: A dictionary with the admin's information
+    """
+    logger.info("Getting admin info")
+
+    admin_info = admins_collection.find_one({"user_id": user_id}, {"_id": 0, "photo": 0})
+    return "\nІнформація: " + json.dumps(admin_info, sort_keys=False, ensure_ascii=False, indent=4)
+
+
+def retrieve_admin_photo(user_id):
+    """
+    EXPERIMENTAL FUNCTION
+    """
+    logger.info("Getting admin photo")
+
+    admin_info = admins_collection.find_one({"user_id": user_id}, {"_id": 0, "photo": 1})
+    return admin_info["photo"]
+
+
+def get_banned_info(user_id):
+    """
+    The get_banned_info function returns a dictionary of information about the user_id that is passed to it.
+
+    :param user_id: Find the user in the blacklist database
+    :return: The information about the user who has been banned
+    """
+    logger.info("Getting banned info")
+
+    banned_info = blacklist.find_one({"user_id": user_id}, {"_id": 0})
+    return "\nІнформація: " + json.dumps(banned_info, sort_keys=False, ensure_ascii=False, indent=4)
```

Comment:
I recommend saying that this info is about bans.
```suggestion
    return "\nІнформація з блекліста: " + json.dumps(banned_info, sort_keys=False, ensure_ascii=False, indent=4)
```
Or something like that.

Reply:
It's actually not necessary